game/script-topics.rpy

```diff
@@ -1883,6 +1883,79 @@ label talk_work_life_balance:
 
     return
 
+# Natsuki discusses her dislike of the horror genre
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._topic_database,
+            label="talk_thoughts_on_horror",
+            unlocked=True,
+            prompt="Thoughts on horror",
+            conditional=None,
+            category=["Natsuki", "Media", "Literature"],
+            nat_says=True,
+            affinity_range=(jn_affinity.RUINED, jn_affinity.LOVE),
+            location="classroom"
+        ),
+        topic_group=TOPIC_TYPE_NORMAL
+    )
+
+label talk_thoughts_on_horror:
+    if jn_affinity.get_affinity_state() <= jn_affinity.NORMAL:
+        n "You know,{w=0.1} [player]..."
+        n "I don't think I ever actually explained why I dislike horror so much."
+        n "I know I mentioned it before,{w=0.1} but I was kinda caught off guard at the time."
+        n "Honestly?"
+        n "Everyone has their tastes,{w=0.1} right? And I can get why people enjoy it."
+
+    elif jn_affinity.get_affinity_state() <= jn_affinity.DISTRESSED:
+        n "You know, I don't think I explained why I dislike horror."
+        n "I get everyone has their tastes, but it isn't for me."
+
+    elif jn_affinity.get_affinity_state() <= jn_affinity.RUINED:
+        n "...I was about to share some of my thoughts on horror with you."
+        n "Or at least,{w=0.1} I was thinking about it."
+        n "...But then do you know what I realized,{w=0.1} [player]?"
+        n "I hate horror -{w=0.1} not that you'd care -{w=0.1} and honestly?"
+        n "...I'm starting to think being here with you is horror enough.{w=0.2} Heh."
+        return
```

Comment:
I think these are meant to be swapped. Unless I'm just dumb, which is quite likely in fact, the second and third `if` statements will never be called, no?

Reply:
You're right, was basically big dum and messed up my conditioning - corrected in next commit